tests/integration/test_interaction_assessment.py

```diff
@@ -148,6 +158,42 @@ def test_max_attempts_reached_submit_and_reset_disabled(self):
         self.assertEqual(submit_button.get_attribute('disabled'), 'true')
         self.assertEqual(reset_button.get_attribute('disabled'), 'true')
 
+    def test_max_attempts_reached_show_answer(self):
+        """
+        Test "Show Answer" button is shown in assessment mode, enabled when no
+        more attempts remaining, and displays correct answers when clicked.
+        """
+        self.place_item(0, TOP_ZONE_ID)
+        show_answer_button = self._get_show_answer_button()
+        self.assertTrue(show_answer_button.is_displayed())
```

Comment:
@arbrandes The "Show Answer" should probably be displayed right away, without having to place an item first? If so, it would make sense to do the check first (i.e., to move the `self.place_item(0, TOP_ZONE_ID)` action from L166 below the `self.assertTrue(show_answer_button.is_displayed())` check).